test/powershell/Modules/Microsoft.PowerShell.Security/ExecutionPolicy.Tests.ps1

```diff
@@ -532,10 +532,10 @@ ZoneId=$FileType
 
         Context "Prereq: Validate that 'Microsoft.PowerShell.Archive' is signed" {
             It "'Microsoft.PowerShell.Archive' should have a signature" {
-                $script:archiveAllCert | Should -Not -Be null
+                $script:archiveAllCert | Should -Not -Be $null
             }
             It "'Microsoft.PowerShell.Archive' should have a valid signature" {
-                $script:archiveCert | Should -Not -Be null
+                $script:archiveCert | Should -Not -Be $null
```

Comment:
Same here, can you do `Should -Not -BeNull` here?

Reply:
Documentation for [BeNullOrEmpty](https://pester.dev/docs/assertions/#benullorempty) states it does a string comparison for null or empty, but `$script:archiveCert` is expected to be of type `System.Security.Cryptography.X509Certificates.X509Certificate2[]`.

Since the array goes through the pipeline and `X509Certificate2` has a `ToString` method this would work as expected, but I think what we should be testing is whether the _array_ is null or empty.

Reply:
I see, thanks for the clarification. Then I guess we should go with `Should -Not -Be $null` as you have changed.